### PR TITLE
footer mobile view

### PR DESCRIPTION
**Pull request by dennisarimi**
Footer views for laptop (same in tablet) and mobile
Screenshot 2024-01-09 at 5 14 07 PM
Screenshot 2024-01-09 at 5 14 55 PM

### CODE REVIEW

**Comment by dennisarimi** [comment]
Footer views for laptop (same in tablet) and mobile
Screenshot 2024-01-09 at 5 14 07 PM
Screenshot 2024-01-09 at 5 14 55 PM

**Comment by dennisarimi** [comment]
Resized window transitions where large mobile 425px < **window** < tablet 768px
Screenshot 2024-01-09 at 5 18 49 PM
Screenshot 2024-01-09 at 5 19 10 PM

File: src/components/Footer/styles.module.css [hunk 1]
```diff
@@ -10,13 +10,8 @@
...
 	margin: auto;
-	padding: 30px;
-}
-
-.links {
-	display: flex;
-	align-items: center;
-	justify-content: space-around;
+	padding-right: 30px;
+	padding-left: 30px;
```
Comment (marksigilai) [inline src/components/Footer/styles.module.css:14]: probably can remove this so it aligns with the rest of the content in the page

**Review by marksigilai — APPROVED** [review]
🐞

**Pull request merged**